src/data-loaders/RelationalDataLoader.js

```diff
@@ -94,12 +94,40 @@ const convertPostgresSQLResponseToColumnMetaData = (rows) => {
       label: row.name,
       name: row.name,
       type: row.dataTypeID,
-      typeName,
+      typeName: typeName ? typeName.toUpperCase() : 'UNKOWN',
     };
   });
 };
+
+const injectVariables = (statement, variableMap) => {
+  const result = Object.keys(variableMap).reduce((statmnt, key) => {
+    if (
+      variableMap[key] === null ||
+      variableMap[key] === false ||
+      variableMap[key] === true
+    ) {
+      return statmnt.replace(key, `${variableMap[key]}`);
+    }
+    // @TODO: Differentiate number from string inputs...
+    return statmnt.replace(key, `'${variableMap[key]}'`);
+  }, statement);
+  return result;
+};
+
 const executeSqlStatements = async (client, req) =>
-  Promise.mapSeries(req.statements, (statement) => client.query(statement));
+  Promise.mapSeries(req.statements, async (statement) => {
+    if (req.variableMap) {
+      statement = injectVariables(statement, req.variableMap);
+    }
+    try {
+      const result = await client.query(statement);
+      return result;
+    } catch (error) {
+      console.log(`RDS_DATALOADER: Failed to execute: `, statement, error);
+      throw e;
```

Comment:
Hi, in this case `e` is undefined. Please replace either `catch (e)` or `throw error`